pkg/volume/local/local.go

```diff
@@ -169,6 +169,7 @@ func (plugin *localVolumePlugin) NewBlockVolumeUnmapper(volName string,
 
 // TODO: check if no path and no topology constraints are ok
 func (plugin *localVolumePlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.Spec, error) {
+	fs := v1.PersistentVolumeFilesystem
```

Comment:
I think CSI volume driver also has the same issue.  Could you update the csi plugin too?

@saad-ali this is another instance of https://github.com/kubernetes/kubernetes/issues/52820

Reply:
Can you also add a unit test for this?

Reply:
@msau42 unit test added.